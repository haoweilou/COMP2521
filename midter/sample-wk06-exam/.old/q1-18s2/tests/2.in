3
4 3 10 5 3 1 7 3 4 1 

