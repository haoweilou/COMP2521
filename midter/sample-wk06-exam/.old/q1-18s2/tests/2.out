Value: 3
Input sequence: 4, 3, 10, 5, 3, 1, 7, 3, 4, 1
After removal: 4, 10, 5, 1, 7, 4, 1
